Dmd/KomplekssSkaitlis.cpp

```diff
@@ -0,0 +1,22 @@
+#include "KomplekssSkaitlis.h"
+#include <iostream>
+using namespace std;
+
+KomplekssSkaitlis::KomplekssSkaitlis(double Number1, double Number2) {
+	number1 = Number1;
+	number2 = Number2;
+}
+
+void KomplekssSkaitlis::Pieskaitit(double number3, double number4) {
+	double sum;
+	sum = (number1 + number3) + (number2 + number4);
+};
```

Comment:
Tev vajadzētu ar to summu kaut ko darīt pēc tam, kad esi aprēķinājusi.
Vai nu atgriest viņu no metodes, vai arī kaut kur saglabāt.
Man liekas uzdevumu noteikumos teikts, ka vajag viņu saglabāt. Šeit tu viņu aprēķini un tad rezultāts pazūd.
Visticamāk tā doma ir ka tu pieskaiti esošajam kompleksajam skaitlim šo jauno komplekso skaitli un tad saglabā pa virsu vecajam skaitlim.